scripts/Muon/GUI/Common/home_plot_widget/home_plot_widget_model.py

```diff
@@ -17,11 +17,46 @@ def __init__(self):
         :param plotting_window_model: This is the plotting manager class to use
         """
         self.plot_figure = None
-        self.plotted_workspaces = []
-        self.plotted_workspaces_inverse_binning = {}
-        self.plotted_fit_workspaces = []
+        self._plotted_workspaces = []
+        self._plotted_workspaces_inverse_binning = {}
+        self._plotted_fit_workspaces = []
         self.plotted_group = ''
 
+    @property
+    def plotted_workspaces(self):
+        """
+        This property is need to check whether all the workspaces contained in the list are still on the graph. They can
+        be removed from the graph from the figure window without this class knowing.
+        :return:
+        """
+        self._plotted_workspaces = [item for item in self._plotted_workspaces if item in
+                                    self.plot_figure.gca().tracked_workspaces.keys()]
+        return self._plotted_workspaces
+
+    @property
+    def plotted_workspaces_inverse_binning(self):
+        self._plotted_workspaces_inverse_binning = {key: item for key, item in self._plotted_workspaces_inverse_binning.items()
```

Comment:
what do you mean by inverse binning?

Reply:
This is a list of binned workspaces if currently plotting unbinned and vice versa. It is used to ensure that fits done on the binned or unbinned data are plotted if either is displayed on the graph.

Reply:
I didn't realise it was to do with binning, I thought it was something to do with 1/(bin width). Could we call it something else to make it clearer?

Reply:
This is in code which went in in #26299 so I don't think here is the right place to fix it. I'll make an issue to update the naming here in future though.